services/elasticache/generator.yaml

```diff
@@ -10,6 +10,11 @@ resources:
         - InvalidParameter
         - InvalidParameterValue
         - InvalidParameterCombination
+    status_fields:
+      - operation_id: ListAllowedNodeTypeModifications
+        member_name: ScaleUpModifications
```

Comment:
How about calling the field `AllowedScaleUpModifications` and `AllowedScaleDownModifications`? Makes the field name more descriptive that the field is advisory and not a part of the ReplicationGroup resource's entity state.

Reply:
Will do it as a follow up patch :) and rename it `AllowedScaleUpModifications` and `AllowedScaleDownModifications`